docs/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Rosa Burner Docs
+---
+
+## What is rosa-burner
+
+Rosa-burner is a wrapper to automate create-use-destroy of OCP clusters in the different managed platforms.
+
+## How works rosa-burner
+
+Following helpers are available for every platform:
+- Schedulers:
+  - Install:
+  - Execute Workloads
+  - Cleanup
+
+Every platform needs to overload following functions to be integrated in the cluster:
+- Initialize
+- Create Cluster
+- Delete Cluster
+- Platform Cleanup
+- Watcher
+
+## Available Platforms
+
+Following platforms are available at this moment:
+
+- Rosa
```

Comment:
This is a bit confusing
![image](https://github.com/cloud-bulldozer/rosa-burner/assets/4614641/cfa943e0-6331-4735-8e1e-b9880693324a)

Reply:
modified